src/renderer/components/dock/create-resource.tsx

```diff
@@ -67,6 +119,23 @@ export class CreateResource extends React.Component<Props> {
     return successMessage;
   };
 
+  renderControls(){
+    return (
+      <div className="flex gaps align-center">
+        <span>Template</span>
+        <Select
+          autoConvertOptions = {false}
+          className="TemplateSelect"
+          options={this.templates}
+          menuPlacement="top"
+          themeName="outlined"
+          onChange={v => this.onSelectTemplate(v)}
+        />
```

Comment:
```suggestion
        <Select
          autoConvertOptions = {false}
          className="TemplateSelect"
          placeholder="Select Template ..."
          options={this.templates}
          menuPlacement="top"
          themeName="outlined"
          onChange={v => this.onSelectTemplate(v)}
        />
```

Reply:
I just took from [here](https://github.com/lensapp/lens/blob/a9a5766920dc055bdd8e27034fa427728ab1da49/src/renderer/components/dock/install-chart.tsx#L152) 
Should I fix selects here?